portal-impl/src/content/Language_bg.properties

```diff
@@ -3876,6 +3876,7 @@ request-a-new-password-reset-link=Поиска нова парола възст
 request-holiday=Заявяване на почивен ден
 request-is-larger-than-x-and-could-not-be-processed=Request is larger than {0} and could not be processed. (Automatic Copy)
 request-pending=Поискат Чакащо (Automatic Translation)
+request-social-networking-summary-add-connection={0} would like to add you as a connection. (Automatic Copy)
```

Comment:
@yuhai It looks like we're losing the translations from the `contacts-web` module (see https://github.com/drewbrokke/liferay-portal/pull/466/commits/16956ff451b5f8a574f856f23a0fb48bfc877fa6#diff-752e8fe1f63c8cec958cb6eed59cf6aaL33).  In this case it's probably best to do a manual copy of the keys from `contacts-web` to `portal-impl`.